src/actions/editProfile.action.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import toastr from 'toastr';
+import Cookie from 'cookies-js';
+import jwt from 'jsonwebtoken';
+
+import config from '../config';
+import { EDIT_PROFILE_BEGINS, EDIT_PROFILE_SUCCESS, EDIT_PROFILE_FAILURE } from './types';
+
+const editProfile = user => (dispatch) => {
+  const token = Cookie.get('jwtToken');
+  const { username } = jwt.decode(Cookie.get('jwtToken'));
+  dispatch({
+    type: EDIT_PROFILE_BEGINS,
+  });
+  return axios.put(`${config.apiUrl}/profiles/${username}`, user, {
+    Headers: {
```

Comment:
I think we already have an interceptor that takes care of setting the authorization token to the request header.
You don't need to do this here.